<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_name">
      <attribute name="title">
       <string>Name Check</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="lay_turtle_node">
         <item>
          <widget class="QCheckBox" name="cb_check">
           <property name="text">
            <string>Check</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_auto_fix">
           <property name="text">
            <string>Auto fix</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_state">
           <property name="styleSheet">
            <string notr="true">border-top: 3px ;
border-bottom: 3px ; 
border-right: 10px ;
border-left: 10px ;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_check">
           <property name="text">
            <string>Check</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_fix">
           <property name="text">
            <string>Fix</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="line_comment">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(192, 192, 192);
background-color: rgba(255, 255, 255, 0);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_show">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(64, 64, 64);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="gb_editAttr">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="box_details">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_box">
              <property name="font">
               <font>
                <family>Corbel</family>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Details</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
              <property name="indent">
               <number>10</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_hide_box">
              <property name="minimumSize">
               <size>
                <width>15</width>
                <height>15</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>15</width>
                <height>15</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_close_box">
              <property name="minimumSize">
               <size>
                <width>15</width>
                <height>15</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>15</width>
                <height>15</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(64, 64, 64);
color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="gb_editing">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="leftMargin">
              <number>4</number>
             </property>
             <property name="topMargin">
              <number>2</number>
             </property>
             <property name="rightMargin">
              <number>4</number>
             </property>
             <property name="bottomMargin">
              <number>2</number>
             </property>
             <item>
              <widget class="QListWidget" name="ls_details"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_help">
      <attribute name="title">
       <string>Help</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="lbl_help">
         <property name="text">
          <string>Help Text</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_close_box</sender>
   <signal>clicked()</signal>
   <receiver>gb_editing</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>438</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>443</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_hide_box</sender>
   <signal>clicked()</signal>
   <receiver>gb_editing</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>443</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_show</sender>
   <signal>clicked()</signal>
   <receiver>gb_editAttr</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
